From 870288a092ba7f689f5522957881ea8fe75adbc8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Tue, 21 May 2024 17:48:06 +0200
Subject: [PATCH 0847/1695] dts: stm32: enable SAES on stm32mp215f-dk
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Sets SAES peripheral to status okay.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 98667e283c990f37b4a62fe1da72874fc4c7e525 dts: stm32: enable SAES on stm32mp215f-dk]
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Change-Id: I1adfa6eb3a3f5f8ed45847235feb1d80aa6545a4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/381352
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/arch/arm/dts/stm32mp215f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp215f-dk.dts b/core/arch/arm/dts/stm32mp215f-dk.dts
index 5f0ddc4e9..1ff8945d2 100644
--- a/core/arch/arm/dts/stm32mp215f-dk.dts
+++ b/core/arch/arm/dts/stm32mp215f-dk.dts
@@ -75,3 +75,7 @@
 		};
 	};
 };
+
+&saes {
+	status = "okay";
+};
-- 
2.47.2

